packages/core/src/routes/resource.ts

```diff
@@ -82,12 +83,21 @@ export default function resourceRoutes<T extends AuthedRouter>(
       // Use the dedicated API `PATCH /resources/:id/is-default` to update.
       body: Resources.createGuard.omit({ id: true, isDefault: true }),
       response: Resources.guard.extend({ scopes: Scopes.guard.array().optional() }),
-      status: [201, 422],
+      status: [201, 400, 422],
```

Comment:
1. this is a breaking change for api. we should guard for frontend only
2. is it RFC requirement?

Reply:
1. Reverted api change
2. Yes, it is. https://datatracker.ietf.org/doc/html/rfc8707#section-2